### PR TITLE
Libtk205 release 0

### PR DESCRIPTION
**Pull request by tanaya-mankad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[![Build and Test](https://github.com/open205/libtk205/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/open205/libtk205/actions/workflows/build-and-test.yml)
+
+libtk205
+===========
+
+A C++ library autogenerated from the current version of the ASHRAE Standard 205P data exchange specification.
```
Comment (nealkruis) [inline README.md]: We should have a link to toolkit-205 somewhere.

Comment (tanaya-mankad) [reply README.md]: Crap, I meant to do that. On it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+The Standard is intended to support the following use cases:
+
+- **Data Publication** Data publishers (typically equipment manufacturers) use representation specifications to guide implementation of data writing and validity testing software that produces correctly-formed representation files.
+
+- **Application Development** Application developers use representation specifications to guide implementation of software that correctly reads representation data. Such implementations may include validity tests and developers may use representation specification example data for testing purposes.
+
+- **Data Application** Application users use representation specifications to understand and check representation data. Data exchange will generally be automated but the availability of representation specifications facilitates additional data checking when needed.
+
+Generally, a data publisher (e.g., manufacturer) provides an ASHRAE Standard 205 representation of a specific piece of equipment that the application user can load into compliant performance simulation software.
```
Comment (nealkruis) [inline README.md]: Maybe we can remove this in favor of a link to http://data.ashrae.org/standard205/?

**Review by nealkruis — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tanaya-mankad) [reply README.md]: Aw, shucks, how can I meet my 1000-word minimum then?

**Review by nealkruis — APPROVED** [review]

**Pull request merged**